Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -0,0 +1,215 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// Controller responsible for displaying the channel messages.
+@available(iOSApplicationExtension, unavailable)
+open class ChatChannelVC: _ViewController, ThemeProvider {
```

Comment:
Is there any reason we are going for `_ViewController` naming instead of private/public `ViewController` or public suffixed `StreamViewController`?

When user will have multiple ViewController namings, they will resolve correctly due they are placed in separate modules, and you always can specify by module name: StreamChat.ViewController and OtherModule.ViewController and make any typealias for it.

Reply:
https://github.com/raywenderlich/swift-style-guide#class-prefixes

Reply:
I remember the reason was that the customers should not be using `_ViewController` or at least should be careful when using it. I'm not sure maybe @b-onc can clarify. I would also prefer naming it just `ViewController` or `StreamViewController`. Either way, I believe this is out of scope of the PR, and all these overall improvements to the SDK can be discussed later 👍